sys-libs/musl/musl-1.2.3-r8.ebuild

```diff
@@ -56,6 +56,10 @@ else
 	PDEPEND="!crypt? ( sys-libs/libxcrypt[system] )"
 fi
 
+PATCHES=(
```

Comment:
We need new revisions for these as it affects install files. I would normally say do not touch old ebuilds, especially for a critical thing like libc, and do it in ~arch, _but_ this is a pretty simple change and it's going to fix some build problems, so it is OK as-is. Just git mv each ebuild please.